engine/core/pom.xml

```diff
@@ -7,6 +8,19 @@
 	</parent>
 	<artifactId>DataCleaner-engine-core</artifactId>
 
+	<repositories>
+		<repository>
+			<id>elasticsearch-releases</id>
+			<url>http://maven.elasticsearch.org/releases</url>
```

Comment:
We're not allowed to add extra repositories to our POM (will mean our deployment to central repo goes haywire). Is this needed?